linkml_runtime/utils/schemaview.py

```diff
@@ -1651,7 +1651,7 @@ def get_identifier_slot(self, cn: CLASS_NAME, use_key: bool = False, imports: bo
         :param imports:
         :return: slot that acts as identifier, if present
         """
-        for sn in self.class_slots(cn, imports=imports):
+        for sn in sorted(self.class_slots(cn, imports=imports)):
```

Comment:
Allows the `identifier` (or `key`) slot to be predicted in cases where there are several slots marked as ID/key. Of course, the user will already have run the linter to eliminate the possibility of there being several ID/key slots...